docs/content/preview/migrate/yb-voyager/yb-voyager-cli.md

```diff
@@ -0,0 +1,146 @@
+description: YugabyteDB Voyager CLI and SSL connectivity.
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  preview:
+    identifier: yb-voyager-cli
+    parent: yb-voyager
+    weight: 105
+isTocNested: true
+showAsideToc: true
+---
+
+yb-voyager is a command line executable program that supports migrating databases from PostgreSQL, Oracle, and MySQL to a YugabyteDB database.
+
+## Syntax
+
+```sh
+yb_voyager [ <migration-phase>... ] [ <arguments> ... ]
+```
+
+- *migration-phase*: See [Migration phase](#migration-phase)
+- *arguments*: See [Arguments](#arguments)
+
+### Command line help
+
+To display the available online help, run:
+
+```sh
+yb_voyager --help
+```
+
+### Migration phase
+
+The following command line options specify the migration phases.
+
+- [export schema](/preview/migrate/yb-voyager/perform-migration/#export-schema)
+
+- [analyze-schema](/preview/migrate/yb-voyager/perform-migration/#analyze-schema)
+
+- [export data](/preview/migrate/yb-voyager/perform-migration/#export-data)
+
+- [import schema](/preview/migrate/yb-voyager/perform-migration/#import-schema)
+
+- [import data](/preview/migrate/yb-voyager/perform-migration/#import-data)
+
+- [import data file](/preview/migrate/yb-voyager/perform-migration/#import-data-file)
+
+### Arguments
+
+#### --export-dir
+
+Specifies the path to the directory containing the data files to export.
+
+#### --source-db-type
+
+Specifies the source database type (PostrgreSQL, Mysql or Oracle).
+
+#### --source-db-host
+
+Specifies the host name of the machine on which the source database server is running.
+
+#### --source-db-user
+
+Specifies the username of the source database.
+
+#### --source-db-password
+
+Specifies the password of the source database.
+
+#### --source-db-name
+
+Specifies the name of the source database.
+
+#### --source-db-schema
+
+Specifies the schema of the source database.
+
+#### --output-format
+
+Specifies the format in which the report file is generated. It can be in `html`, `txt`, `json` or `xml`.
+
+#### --target-db-type
+
+Specifies the target YugabyteDB database between [YugabyteDB](https://www.yugabyte.com/yugabytedb/), [YugabyteDB Anywhere](https://www.yugabyte.com/anywhere/) or [YugabyteDB Managed](https://www.yugabyte.com/managed/).
+
+#### --target-db-host
+
+Specifies the host name of the machine on which target database server is running.
+
+#### --target-db-user
+
+Specifies the username of the source database.
+
+#### --target-db-password
+
+Specifies the password of the source database.
+
+#### --target-db-name
+
+Specifies the name of the source database.
+
+#### –-data-dir
+
+The path to the directory containing the data files to import.
+
+#### --file-table-map
+
+Comma-separated mapping between the file in [data-dir](#–-data-dir) to the corresponding table in the database.
+Default : The file name in `table1_data.sql` format where `table1` name will be used as the table name.
```

Comment:
This has changed. Now default is `.csv` files (instead of the `.sql` files).

By default, `yb-voyager import data file` command imports files (from the `--data-dir` directory) that match the `PREFIX_data.csv` pattern. The `PREFIX` part is considered as table-name into which the file is imported.